apps/api/src/.env.test

```
@@ -41,7 +41,7 @@ SENDGRID_API_KEY=SG.123123
 S3_ACCESS_KEY=
 S3_SECRET=
 REDIS_ARENA_PORT=4568
-FRONT_BASE_URL=http://127.0.0.1:4200
+FRONT_BASE_URL='^https?://([a-zA-Z-_0-9]+.)?(dashboard(-v[0-9]+)?.novu(-staging)?.co)$'
```

Should we rename the name to something more fitting? 

maybe
```suggestion
FRONT_BASE_URL_REGEX='^https?://([a-zA-Z-_0-9]+.)?(dashboard(-v[0-9]+)?.novu(-staging)?.co)$'
```

---

@djabarovgeorge this env variable has 2 purposes now, the regexp for Novu cloud and the url for the os

---

thanks for the feedback, then lets keep it like this :)